releaseNotes/PythonFeatureManagement.md

```diff
@@ -0,0 +1,19 @@
+# featuremanagement
+
+[Source code][source_code] | [Samples][samples]
+
+## 1.0.0b1 - May 22, 2024
+
+Initial Beta Release of the Feature Management for Python
+
+### New Features
+
+* Loading of feature flags from a file, see [feature management schema].
+* Loading of feature flags from Azure App Configuration provider.
+* Checking if a feature is enabled.
+* Default feature filters: TimeWindowFilter, TargetingFilter.
+* Custom feature filters.
```

Comment:
Suggest below:

### Enhancement

Initial release of the feature management support for Python
- Support for loading feature flags from ... (dictionary?).
- Support for basic feature flags with boolean states.
- Support for feature filters including built-in filters `TimeWindowFilter` and `TargetingFilter`.

Reply:
Shouldn't we call out custom filters too?

Reply:
All feature filters are custom filters. It's just that we ship some "custom filters" out of the box as well.